src/components/Calculator.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import calculate from '../logic/calculate';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  clickHandle = (e) => {
+    const clickedBtn = e.target.innerText;
+    const Result = calculate(this.state, clickedBtn);
+    this.setState(Result);
+  }
+
+  render() {
+    const { total, operation, next } = this.state;
+    return (
+      <div className="MainBody">
+        <div className="result">
+          {total}
+          {operation}
+          {next}
+        </div>
+        <div className="num-symb">
+          <button className="buttons" type="button" onClick={this.clickHandle}>AC</button>
+          <button className="buttons" type="button" onClick={this.clickHandle}>+/-</button>
+          <button className="buttons" type="button" onClick={this.clickHandle}>%</button>
+          <button className="buttons operator" type="button" onClick={this.clickHandle}>÷</button>
```

Comment:
- [ ] Great job so far however, all math operations should have a specific click even. However you're missing the division operator (÷), Kindly make sure to implement this 👍🏻